src/fetcher.js

```diff
@@ -31,6 +32,8 @@ export function handler (input: ?any, context: ?any, callback: (?Error, ?output)
   })
 }
 async function asyncHandler (input: input): Promise<Array<result>> { // eslint-disable-line no-use-before-define
+  console.log('Pausing for 120 seconds')
+  await sleep(120*1000); // 120 seconds
```

Comment:
We don't need to do this, we can configure it all in the state machine

Reply:
Updated. Thanks for that, sooo much better!